dev/com.ibm.ws.jaxrs.2.0.cdi/src/com/ibm/ws/jaxrs20/cdi/component/JaxRsFactoryImplicitBeanCDICustomizer.java

```diff
@@ -50,6 +50,7 @@
 import com.ibm.websphere.ras.TraceComponent;
 import com.ibm.websphere.ras.annotation.Trivial;
 import com.ibm.ws.cdi.CDIService;
+import com.ibm.ws.cdi.internal.interfaces.CDIUtils;
```

Comment:
It seems weird to be importing an "internal" package.  Please make sure that the CDI team is ok with us using this class - possibly a better approach would be to put CDI Utils in a different (more public) package.

Reply:
Ben's new PR https://github.com/OpenLiberty/open-liberty/pull/10690